InvenTree/stock/templates/stock/low_stock_notification.html

```diff
@@ -0,0 +1,27 @@
+<p>{{ message }}</p>
```

Comment:
Rather than passing `{{ message }}` through to the template, we should construct the message in the template file itself.

Reply:
I changed the same in my latest commit based on the suggestions my @matmair .
Kindly refer the [commit](https://github.com/inventree/InvenTree/pull/2208/commits/6ec2801fcea15d0784e0ad57f6000e8b568aa05b)